.github/workflows/condabuild.yml

```diff
@@ -0,0 +1,88 @@
+---
+name: Conda
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      gradle_version:  # set to empty to build with most recent version of gradle
+      gradle_commands: publishToMavenLocal exec # default is build
```

Comment:
```suggestion
    env:
      gradle_version:  # set to empty to build with most recent version of gradle
      gradle_commands: publishToMavenLocal exec # default is build
```

Reply:
not possible to commit the suggestion via the UI (same text). do you want to push directly to my branch?

Reply:
Oops, I meant to delete the lines not copy them 😄 .

Reply:
done